test/unit/response.test.js

```diff
@@ -786,6 +820,48 @@ describe('Response', function () {
             });
         });
 
+        describe('timings', function () {
+            it('should not include timing information', function (done) {
```

Comment:
specify the reason/condition which leads to this assertion.
```suggestion
            it('should not include timing information when time is false', function (done) {
```